pkg/daemon/ceph/client/image.go

```diff
@@ -116,10 +133,13 @@ func CreateImage(context *clusterd.Context, clusterInfo *ClusterInfo, name, pool
 	return &CephBlockImage{Name: name, Size: newSizeBytes}, nil
 }
 
-func DeleteImage(context *clusterd.Context, clusterInfo *ClusterInfo, name, poolName string) error {
+func DeleteImage(context *clusterd.Context, clusterInfo *ClusterInfo, name, poolName, namespace string) error {
```

Comment:
Similarly, I'd suggest `DeleteImageInPool()` and `DeleteImageInRadosNamespace()`